db/migrate/20220702123618_create_metais.rb

```diff
@@ -0,0 +1,75 @@
+class CreateMetais < ActiveRecord::Migration[6.0]
+  def up
+    execute 'CREATE SCHEMA metais'
```

Comment:
Tu este bude treba pridelit opravnenia role opendata, aby to ludia vedeli citat cez sql api.